src/components/Navigation/Navigation.js

```diff
@@ -118,22 +117,21 @@ const Navigation = () => {
         setMobileOpen((open) => !open);
     }, []);
 
+    const matches = useMediaQuery('(margin-left: 233px)');
+    console.log(useMediaQuery("margin-left:233px"))
+
     return (
         <ThemeProvider theme={OmouTheme}>
             <div className="Navigation">
-                <AppBar className="OmouBar" position="sticky">
-                    <Toolbar>
-                        {token ?
-                            <AuthenticatedNav
-                                toggleDrawer={handleDrawerToggle} /> :
-                            <UnauthenticatedNav />}
-                    </Toolbar>
-                </AppBar>
+                {token ?
+                    <AuthenticatedNav
+                        toggleDrawer={handleDrawerToggle} /> :
+                    <UnauthenticatedNav />}
```

Comment:
Without `<UnathenticatedNav>` the code should just look like: `token && <AuthenticatedNav toggleDrawer={...}/>`